WooCommerce/Resources/en.lproj/Localizable.strings

```diff
@@ -801,6 +801,9 @@
 /* Legal disclaimer for signup buttons, the underscores _..._ denote underline */
 "By signing up, you agree to our _Terms of Service_." = "By signing up, you agree to our _Terms of Service_.";
 
+/* Title for the helper field describing calculated amount for given percentage */
+"Calculated amount" = "Calculated amount";
```

Comment:
Found in the version with the new tooling, too. See [`5c21528`](https://github.com/woocommerce/woocommerce-ios/pull/6737/commits/5c215284d6839f397732f684ec664405f42939ab#diff-96d03942a96e588fcdca0288a2a18d0466a6e44ee5b256f581cd17f2086ea22dR780-R781)